CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 * Fixed an issue where `ReplayLocationManager` would crash if initialized with just one location. ([#3528](https://github.com/mapbox/mapbox-navigation-ios/pull/3528))
 * Added the `ReplayLocationManager.onReplayLoopCompleted` property that allows you to stop the location manager from looping back to the beginning of the route. ([#3528](https://github.com/mapbox/mapbox-navigation-ios/pull/3528))
 
+### Banners and guidance instructions
+
+* Fixed the crash when scrolling the guidance cards while the orienttaion change during active navigation. ([#3544](https://github.com/mapbox/mapbox-navigation-ios/pull/3544))
```

Comment:
Typo in "orienttaion". 

nit: This changelog entry is hard to read, I would suggest to simplify it.